store/src/main/java/com/nytimes/android/external/store3/base/impl/RealInternalStore.java

```diff
@@ -211,15 +212,21 @@ void notifySubscribers(Parsed data) {
     @Nonnull
     @Override
     public Observable<Parsed> stream(@Nonnull Key key) {
```

Comment:
Might the `stream` name that return an `Observable` cause a little bit of confusion with Java 8 `Stream` API?